src/features/analytics/AnalyticsPage.tsx

```diff
@@ -48,215 +65,279 @@ function AnalyticsPage() {
-              },
-              Footer: (row: FooterTable<AnalyticsSalaryForSeo>) => getTotalCost(row, 'retainer'),
-            },
-            {
-              Header: 'Parking Cost (per Month)',
-              accessor: 'parkingCostPerMonth',
-              disableSortBy: true,
-              ...getSelectFilterOptions('parkingCostPerMonth', true),
-              Cell: ({ row }: CellTable<AnalyticsSalaryForSeo>) => {
-                const { parkingCostPerMonth } = row.original;
-
-                return (<div>{formatMoney(parkingCostPerMonth)}</div>);
-              },
-              Footer: (row: FooterTable<AnalyticsSalaryForSeo>) => getTotalCost(row, 'parkingCostPerMonth'),
-            },
           ]}
         />
       </div>
 
     </ContentCard>
   );
 
-  async function loadEmployeesAsync() {
-    setIsLoading(true);
-
-    try {
-      const { data } = await api.get<AnalyticsSalaryForSeo[]>('/finances/get-finance-data');
+  function getTotalCost(total: number, sumTotal: number, isPrecent: boolean = false) {
+    sumTotal = Number(sumTotal.toFixed(2));
+    total = Number(total.toFixed(2));
+    return (
+      <div>
+        <RedactComponent
+          value={(isPrecent ? `${sumTotal}%` : formatMoney(sumTotal))}
+          valueDelta={Number((sumTotal - total).toFixed(2))}
+        />
+      </div>
+    );
+  }
 
-      setEmployees(data);
+  function dublicateEmployee(idEmployee: number) {
+    const copyEmployee = employees.find((el) => el.id === idEmployee);
+    if (copyEmployee) {
+      setEmployees([...employees, copyEmployee]);
+    }
+  }
 
-      setIsLoading(false);
-    } catch {
-      // todo: #344qg6j
-      setIsLoading(false);
+  async function deleteEmployee(idEmployee: number) {
+    const copyEmployee = employees.find((el) => el.id === idEmployee);
+    if (copyEmployee) {
+      const index = employees.indexOf(copyEmployee);
+      const newEmployees = employees.slice();
+      newEmployees.splice(index, 1);
+      setEmployees(newEmployees);
     }
   }
 
-  function getTotalCost(row: FooterTable<AnalyticsSalaryForSeo>, keyOfEmployee: string) {
-    const { filteredRows } = row;
+  async function updateEmployeesAsync(employee: PutPreviewType) {
+    const { data } = await api.post<GetPreviewType>('finance/get-preview', employee);
 
-    const sumOfEmployeesValues = filteredRows.map((elem) => elem.values[keyOfEmployee as keyof AnalyticsSalaryForSeo] as number)
-      .reduce((pre: number, current: number) => pre + current);
+    const index = employees.find((el) => el.id === employee.employeeId);
 
-    return (
-      <div>{formatMoney(sumOfEmployeesValues)}</div>
-    );
+    if (index) {
+      const newemp = employees.slice();
+      newemp[newemp.indexOf(index)] = data;
+      setEmployees(newemp);
+    }
   }
 
-  function getSelectFilterOptions(keyOfEmployee: string, isFormatNumber: boolean = false) {
-    const all = { label: 'All', value: '' };
+  async function loadTotals() {
```

Comment:
The total is calculated from the data processed by the table itself (this is more convenient, because the table gives the data that have been filtered, sorted and paginated)